ts/torch_handler/base_handler.py

```diff
@@ -194,18 +199,50 @@ def handle(self, data, context):
         self.context = context
         metrics = self.context.metrics
 
-        data_preprocess = self.preprocess(data)
-
-        if not self._is_explain():
-            output = self.inference(data_preprocess)
-            output = self.postprocess(output)
+        is_profiler_enabled = os.environ.get("ENABLE_TORCH_PROFILER", None)
```

Comment:
@shrinath-suresh I wonder if this could be added to the config.properties, this would help to make sure all the configs are gathered in one place. An example of adding to the config can be found [here](https://github.com/pytorch/serve/pull/1319/files).

Reply:
@chauhang your thoughts on this ?

Reply:
I agree with Hamid, it's a simple change that'll make sure people don't need to think about both OS environment variables and config.properties. It's how we manage all configs. Let us know if you have any questions about how this would work